### PR TITLE
Make you not hit the cans with tanks anymore

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request
Ghilker by mistake put callback to parrent at the start of portable atmos attack chain, I moved it 2 lines lower and it works fine now.

## Why It's Good For The Game
Fixes this piece of shit: https://github.com/tgstation/tgstation/issues/58152

## Changelog
:cl: SparkezelPL
fix: You can no longer hit canisters with tanks 
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — APPROVED** [review]

File: code/modules/atmospherics/machinery/portable/portable_atmospherics.dm [hunk 1]
```diff
@@ -133,8 +133,8 @@
 	return TRUE
 
 /obj/machinery/portable_atmospherics/attackby(obj/item/W, mob/user, params)
-	. = ..()
 	if(!istype(W, /obj/item/tank))
+		. = ..()
```
Comment (Fikou) [inline code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: this makes no sense you set return value to ..() and then return false

Comment (Sparkezel) [reply code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: it works though and I see no conflicts,
does `. = ..()` actually return? I was not sure

Comment (LemonInTheDark) [reply code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: oh fuck he's right

Comment (Kylerace) [reply code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: no it doesnt return, it sets the default return value for the proc to parent

Comment (Sparkezel) [reply code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: kk fixing rn

Comment (Fikou) [reply code/modules/atmospherics/machinery/portable/portable_atmospherics.dm:137]: . is the return value of the proc, what is returned if you do return
you set the return value but then immediatelly did a return FALSE so theres no purpose at all in setting it

**Pull request merged**